sentences/en/light_HassLightSet.yaml

```
@@ -3,73 +3,41 @@ intents:
   HassLightSet:
     data:
       # brightness
+      # specific entity control
       - sentences:
-          - "[<numeric_value_set>] <name> brightness [to] <brightness>"
-          - "[<numeric_value_set>] [the] brightness [of] <name> [to] <brightness>"
-          - "[<numeric_value_set>] <name> [to] <brightness> [brightness]"
+          - "[<numeric_value_set>] ([of] <name>;[to] <brightness_value>)"
+          - "[<numeric_value_set>] ([of] <name>;[the] <brightness_synonyms>) [to] <brightness_value>"
+          - "[<numeric_value_set>] <name> [to] <brightness_value> <brightness_synonyms>"
         response: "brightness"
         requires_context:
           domain: light
 
+      # specific area/floor control
       - sentences:
-          - "<numeric_value_set> <name> to <brightness>"
-        requires_context:
-          domain: "light"
-        response: "brightness"
-      - sentences:
-          - "[<numeric_value_set>] [the] brightness in <area> to <brightness>"
-          - "[<numeric_value_set>] [the] brightness of <area> to <brightness>"
-          - "[<numeric_value_set>] <area> brightness [to] <brightness>"
-          - "[<numeric_value_set>] <area> [to] <brightness> brightness"
-          - "[<numeric_value_set>] <area> [to] <brightness>"
-          - "[<numeric_value_set>] [(<all>|<the>)] <light> [<in>] <area> to <brightness> [brightness]"
-        response: "brightness"
-
-      - sentences:
-          - "<numeric_value_set> <area> to <brightness>"
-        response: "brightness"
-
-      - sentences:
-          - "[<numeric_value_set>] [the] brightness to <brightness>"
-          - "[<numeric_value_set>] [the] brightness (<in_here>;to <brightness>)"
+          - "[<numeric_value_set>] ([of] [(<all>|<the>)] <location>;[to] <brightness_value>)"
+          - "[<numeric_value_set>] ([of] [(<all>|<the>)] <location>;[the] <brightness_synonyms>) [to] <brightness_value>"
+          - "[<numeric_value_set>] [(<all>|<the>)] <location> ([to] <brightness_value>;[the] <brightness_synonyms>)"
         expansion_rules:
-          in_here: "[in] here"
+          location: "(<in_area_floor>|(<light>;<in_area_floor>))"
         response: "brightness"
-        requires_context:
-          area:
-            slot: true
 
-      # Max/Min brightness
+      # inferred current area control
       - sentences:
-          - "[<numeric_value_set>] <name> brightness to [the] {brightness_level:brightness}"
-          - "[<numeric_value_set>] [the] brightness of <name> to [the] {brightness_level:brightness}"
-          - "[<numeric_value_set>] <name> [to] [the] {brightness_level:brightness} brightness"
-        requires_context:
-          domain: light
-        response: "brightness"
+          - "[<numeric_value_set>] [<the>] <light> [to] <brightness_value>"
+          - "[<numeric_value_set>] [<the>] <light> ([to] <brightness_value>;<here>)"
+          - "[<numeric_value_set>] [(<all>|<the>)] <light> ([to] <brightness_value>;<here>)"
```

These can be contracted, right?
```suggestion
          - "[<numeric_value_set>] [(<all>|<the>)] <light> ([to] <brightness_value>;<here>)"
```

---

Yep, another good catch. I went ahead and made this change as well as the similar suggestion below.